interface/usergroup/usergroup_admin.php

```diff
@@ -20,15 +20,17 @@
  * @link    http://www.open-emr.org
  */
 
-require_once("../globals.php");
-require_once("../../library/acl.inc");
-require_once("$srcdir/auth.inc");
+require_once "../globals.php";
+require_once "../../library/acl.inc";
+require_once "$srcdir/auth.inc";
+use Symfony\Component\HttpFoundation\Request;
+use services\UserService;
```

Comment:
I couldn't find anything specifically stating `use` has to go above `require` statements, only that `use` must be followed with a blank line (Which this file complies with).

phpstorm didn't throw a warning on this file either